Wireframe/index.html

```diff
@@ -4,10 +4,46 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Wireframe</title>
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="css/style.css">
   </head>
   <body>
-    <!-- Add your HTML markup here -->
+    <header>
+      <h1>Understanding Git</h1>
+      <p>This is all about understanding how git works.</p>
+  </header>
+
+  
+      <article id="what-is-git">
+          <h2>What is Git?</h2>
+          <p>Git is a distributed version control system that helps developers track changes in their code over time. It is widely used to manage project versions and collaborate with other developers efficiently.</p>
+
+          <a href="#">Learn More</a>
+      </article>
+
+      <div class="article2-article3">
+      <div>
+      <article class="section" id="why-developers-need-git">
+          <h2>Why Do Developers Need Git?</h2>
+          <ul>
+              <li><strong>Version Control:</strong> Keep a history of all code changes.</li>
+              <li><strong>Collaboration:</strong> Multiple developers can work on the same project without conflicts.</li>
+              <li><strong>Backup:</strong> Git stores a full copy of the project in each developer's local repository.</li>
+              <li><strong>Branching & Merging:</strong> Work on different features in parallel without disturbing the main codebase.</li>
+              <li><strong>Reverting Changes:</strong> Easily roll back to previous versions if needed.</li>
+          </ul>
+          <a href="#">Learn More</a>
+      </article>
+      </div>
+
+      <div>
+      <article class="section" id="what-is-branch">
+          <h2>What is a Branch in Git?</h2>
+          <p>A branch in Git is a separate workspace within the repository. It allows developers to work on new features, bug fixes, or experiments without affecting the main project. Branches are often merged back into the main project once the work is complete and verified.</p>
+          <p><strong>Example:</strong> A developer may create a 'feature' branch to add new functionality, while others may work on separate branches. Once the new feature is ready, it can be merged into the 'main' branch.</p>
+          <a href="#">Learn More</a>
+      </article>
+      </div>
+      </div>
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
 </body>
```

Comment:
According to the requirements and wireframe, there should be a footer as well!

Reply:
the footer has been the added